src/pipeline.rs

```diff
@@ -78,7 +78,6 @@ pub fn generate_monitor_pipeline(
             ("project", project.to_owned()),
             ("package", package.to_owned()),
             ("repository", repo.to_owned()),
-            ("arch", arch.to_owned()),
```

Comment:
this doesn't actually work, a bunch of commands here do actually need `--arch` so the tests fail:

```
122 tests run: 26 passed, 96 failed, 0 skipped
```

Reply:
Thank you. I tried `cargo test` but the tests got hung for me.


```
... snipped ...

test upload::tests::test_create_list_package ... ok
test handler::tests::test_handler_flow::dput_test_4::build_success_1::log_test_3::download_binaries_1::prune_only_if_job_unsuccessful_2 ... FAILED
test retry::tests::test_retry_on_nested_non_client_errors ... ok
test retry::tests::test_retry_on_non_client_errors ... ok
test handler::tests::test_handler_flow::dput_test_4::build_success_1::log_test_1::download_binaries_1::prune_only_if_job_unsuccessful_1 ... FAILED
test handler::tests::test_handler_flow::dput_test_4::build_success_1::log_test_2::download_binaries_1::prune_only_if_job_unsuccessful_1 ... FAILED

test monitor::tests::test_fails_after_repeated_duplicate_endtimes ... FAILED
test monitor::tests::test_handles_old_build_status has been running for over 60 seconds
test prune::tests::test_prune has been running for over 60 seconds
test retry::tests::test_no_retry_on_client_errors has been running for over 60 seconds
test retry::tests::test_no_retry_on_nested_client_errors has been running for over 60 seconds


^C
```